### ISSUE TITLE
Enhancement: Move message to other room

### ISSUE THREAD

**Issue by penguin42** [vector-im/vector-android#433]
Say you pick your phone up and send something and realise it should be in a different room; a convenient feature would be a 'move' - it would let you select a message (or set of them?) and let you move them to another room.
What it would actually do is issue redact's on the original messages and repost them to the new one.

**Comment by ara4n** [comment]
Yup, this is on the medium-term roadmap as part of our plans for 
threading.  The hope is that you can split the conversation into 
threads, and then chose to move threads to other rooms if you so desire.
